### PR TITLE
Update Profile: Previous Profile data displayed in the Card.

### LINKED ISSUE #779: Display Initial data in update profile card.
For now, when a user opens the update profile view, then he sees blank fields.
What I was thinking that he must see the previous `username`, `first_name`, `last_name` in the card & it must be editable, so that the user can change it.
@deshraj @spyshiv @aka-jain what are your thoughts about it?

### PR DESCRIPTION
**Pull request by RishabhJain2018**
fixes #779 . @deshraj @spyshiv @aka-jain Please review the PR.

### CODE REVIEW

File: frontend/src/js/controllers/updateProfileCtrl.js [hunk 1]
```diff
@@ -33,6 +33,28 @@
...
+            onSuccess: function(response) {
+                var status = response.status;
+                var result = response.data;
+                if (status == 200) {
+                    vm.user = result;
+                }
+
+            },
+            onError: function(response) {
+                $rootScope.notify("error", "Error in loading old profile, please try again.");
```
Comment (deshraj) [inline frontend/src/js/controllers/updateProfileCtrl.js]: Please change the notification message to `Error in loading profile, please try again later!`

**Comment by spyshiv** [comment]
LGTM. :+1:

**Comment by RishabhJain2018** [comment]
@deshraj @spyshiv Please review the PR.

File: frontend/src/js/controllers/updateProfileCtrl.js (hunk 1 above)
Comment (aka-jain) [inline frontend/src/js/controllers/updateProfileCtrl.js]: @RishabhJain2018 Please make this correct.

Comment (RishabhJain2018) [inline frontend/src/js/controllers/updateProfileCtrl.js]: @aka-jain I had made the chenges. Please refer to this [link](https://github.com/Cloud-CV/EvalAI/pull/801/files#diff-6337108b641fb2b83e8b675eaf1572d0R50). :-)

**Pull request merged**